pkgs/applications/office/paperless-ngx/default.nix

```diff
@@ -43,30 +33,16 @@ let
   path = lib.makeBinPath [ ghostscript imagemagick jbig2enc optipng pngquant qpdf tesseract4 unpaper ];
 in
 py.pkgs.pythonPackages.buildPythonApplication rec {
-  pname = "paperless-ng";
-  version = "1.5.0";
+  pname = "paperless-ngx";
+  version = "1.6.0";
 
   src = fetchurl {
-    url = "https://github.com/jonaswinkler/paperless-ng/releases/download/ng-${version}/${pname}-${version}.tar.xz";
-    sha256 = "oVSq0AWksuWC81MF5xiZ6ZbdKKtqqphmL+xIzJLaDMw=";
+    url = "https://github.com/paperless-ngx/paperless-ngx/releases/download/ngx-${version}/${pname}-${version}.tar.xz";
+    sha256 = "07mrxbwahkm00n9nvssd6d13p80w333g84cd38bzp0l34nzim5zl";
   };
 
-  patches = [
-    # Fix the `slow_write_pdf` test:
-    # https://github.com/NixOS/nixpkgs/issues/136626
-    (fetchpatch {
-      url = "https://github.com/paperless-ngx/paperless-ngx/commit/4fbabe43ea12811864e9676b04d82a82b38e799d.patch";
-      sha256 = "sha256-8ULep5aeW3wJAQGy2OEAjFYybELNq1DzCC1uBrZx36I=";
-    })
-  ];
-
   format = "other";
 
-  # Make bind address configurable
-  postPatch = ''
-    substituteInPlace gunicorn.conf.py --replace "bind = '0.0.0.0:8000'" ""
-  '';
-
```

Comment:
Was this removed on purpose? I think the address is fixed to `0.0.0.0` by default

Reply:
Sorry for not explaining this.
Yes, I've removed it because it's not needed (and maybe never was).
You can see that in the NixOS test `paperless.port` is correctly bound, although env var `PAPERLESS_PORT` is never set by the service. This proves that `GUNICORN_CMD_ARGS` overrides the `bind` var in `gunicorn.conf.py`.

You can also check that only localhost is bound, like so:
```bash
config='{
  containers.tmp = {
    extra.addressPrefix = "10.30.0";
    config.services.paperless-ng.enable = true;
  };
}'
# Requires extra-container 0.9
extra-container shell --nixpkgs-path /path/to/nixpkgs -E "$config" --run bash -c 'sleep 1; c netstat -nltp'
# Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name    
# tcp        0      0 127.0.0.1:28981         0.0.0.0:*               LISTEN      422/python3.9       
#                     ^^^^^^^^^
```